proposal.md

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE
+
+## 🔥 Do not edit this file - copy the template and create your own file.
+
+### 📌 Step 1
+
+**TITLE:**
+Intro to Python Libraries: Build an Alexa Skill
+
+**TOPIC:**
+Intro to Python Libraries
+
+**DESCRIPTION (5-7+ sentences):**
+This workshop will have two main sections and focuses on introducing students to different libraries and the directions they could go with a versatile language like python. The workshop will overview core data science libraries like sci-kit learn, pandas, and may go into deep learning like tensorflow and keras to expose students. Web development in python with flask and django will be presented to show a different direction. Finally, I plan to express how you can use python to write scripts to bring your ideas to life. This final point will be implemented by walking the user through building their own roulette game alexa skill. 
+
+​	
+
+
+
+### 📌 Step 2
+
+👪 **TARGET AUDIENCE (3-5+ sentences):**
+The target audience of this workshop will be entry level programmers. They should have understandings of beginner level concepts like functions, lists etc but are looking to add more to their projects. I found that BitProject had many blogs and workshops in the works describing these libraries including an intoduction to pandas, flask and linear regression with scikit learn. This workshop's goal is to provide a description of pythons libraries to give beginners a launching point into one of the more technical blogs while also providing a fun project.
```

Comment:
That is a lot of libraries to teach to entry level programmers. IS this going to be a couple.a series of blogs or is this going to be one long blog?

Reply:
This is currently being developed as a workshop so over viewing these libraries and a technical walkthrough will fit within the time frame.